grails-app/assets/javascripts/reporting.js

```diff
@@ -239,6 +263,13 @@ var ReportViewModel = function(report, config) {
         ko.applyBindings(reasonViewModel, $reasonModal[0]);
         $reasonModal.modal({backdrop: 'static', keyboard:true, show:true}).on('hidden', function() {ko.cleanNode($reasonModal[0])});
     };
+
+    self.outcomeCategory = ko.pureComputed(function() {
```

Comment:
We don't want this to only apply to only outcomes report 1 - either we apply it to all reports, or we add a configuration somewhere to mark a report type as cancellable.  I'd start with applying to all reports.

Reply:
I removed the binding to apply the button to all reports.